helm/oauth2-proxy/Chart.yaml

```diff
@@ -14,7 +14,7 @@ keywords:
   - redis
 dependencies:
   - name: redis
-    version: ~10.6.0
+    version: ~13.0.0
```

Comment:
The latest version of the Bitnami Redis chart is `16.3.1`
https://github.com/bitnami/charts/blob/master/bitnami/redis/Chart.yaml#L27